imports/pages/series/index.js

```diff
@@ -57,9 +57,10 @@ const withSeries = graphql(SERIES_QUERY, {
     loading: data.loading,
     done: (
       data.content &&
-      data.loading && // XXX This is a bug in react-apollo, make !data.loading after fixed
+      !data.loading && // XXX This is a bug in react-apollo, make !data.loading after fixed
       data.content.length < data.variables.limit + data.variables.skip
     ),
+    doneText: "End of Series",
```

Comment:
Instead of defining this here, I'd prefer it gets passed as a property directly on the component. Like:

```
<ApollosPullToRefresh doneText="End of Series">
```

And in fact, I think we could just pass the category name?

```
<ApollosPullToRefresh category="Series">
```

I'm also interested to see if we could bypass this altogether and use the category name helper to dynamically pull the content type, but this would need to be overridden for the Home Feed.

https://github.com/NewSpring/Holtzmann/blob/master/imports/util/categories/categories.name.js